mailbox/noise.go

```diff
@@ -489,10 +497,20 @@ func ekeUnmask(me *btcec.PublicKey, password []byte) *btcec.PublicKey {
 }
 
 const (
+	// HandshakeVersion0 is the handshake version that corresponds to
+	// the handshake where the authData is sent in act two instead of
+	// in act 4.
+	HandshakeVersion0 = byte(0)
+
+	// MinHandshakeVersion is the minimum handshake version that is
+	// currently supported.
+	MinHandshakeVersion = HandshakeVersion0
+
 	// HandshakeVersion is the expected version of the brontide handshake.
-	// Any messages that carry a different version will cause the handshake
+	// Any messages that carry a version that is not between
+	// MinHandshakeVersion and HandshakeVersion will cause the handshake
 	// to abort immediately.
-	HandshakeVersion = byte(0)
+	HandshakeVersion = byte(1)
```

Comment:
Should we define `HandshakeVersion1` above as well, then just point this to that version? And perhaps rename this to `CurrentHandshakeVersion` to make it less confusing naming wise?